Kitodo/src/main/java/org/kitodo/services/data/base/SearchService.java

```diff
@@ -57,29 +58,34 @@
     private static final Logger logger = LogManager.getLogger(SearchService.class);
     protected Searcher searcher;
     protected Indexer indexer;
+    protected BaseType type;
 
     /**
      * Constructor necessary to use searcher in child classes.
      *
      * @param searcher
      *            for executing queries
      */
-    public SearchService(V dao, Searcher searcher) {
+    public SearchService(V dao, BaseType type, Indexer indexer, Searcher searcher) {
```

Comment:
Missing JavaDoc for parameter `dao`, `type` and `indexer`.

Reply:
added